lib/cli/src/generators/WEB-COMPONENTS/template-csf/.storybook/preview.js

```diff
@@ -1,30 +1,3 @@
-/* global window */
-
-import {
-  configure,
-  addParameters,
-  // setCustomElements,
-} from '@storybook/web-components';
-
+// import { setCustomElements } from '@storybook/web-components';
```

Comment:
If there's nothing in the file anymore, then there's no need to add it, right?

However I do think it's important to have preview.js out of the box, maybe use a global parameter like other templates?

Reply:
I'll try to make it line up with the other templates. There is commented code in the file that shows you how to set up your custom-elements.json

Reply:
✅ done. good call!